app/src/main/java/org/p2p/wallet/common/crashlogging/CrashLogger.kt

```diff
@@ -2,6 +2,7 @@ package org.p2p.wallet.common.crashlogging
 
 import org.p2p.wallet.infrastructure.network.provider.TokenKeyProvider
 
+@Suppress("DEPRECATION")
```

Comment:
Please don't touch other classes if you are not working on it. 
These changes are about profile, not logger.

The same with other annotations